metplus/PYTHON_VERSION_MIN

```diff
@@ -0,0 +1 @@
+3.6.3
```

Comment:
Should this be 3.8.6 too? I'm not sure what is going on here but wanted to check.

Reply:
No, this is the minimum version that is required to run METplus wrappers. An error will be thrown if the version is less than this version. A warning will be displayed if the version is above this but below the recommended version.